docs/glossary.md

```diff
@@ -19,21 +19,25 @@ Amphora is the data composer and REST API of Clay. Its core responsibilities inc
 - Discovering each plugin available in an instance of Clay
 - Migrating component data as components iterate over time
 
-<hr/>
+You can get more detail on our [Amphora documentation](https://docs.clayplatform.com/amphora/docs/intro)
```

Comment:
Does changing this to `details` make sense? 
```suggestion
You can get more details on our [Amphora documentation](https://docs.clayplatform.com/amphora/docs/intro)
```

Reply:
Nice catch!! Thanks!!